### ISSUE TITLE
Chat seems to be entirely inaccessible

### ISSUE THREAD

**Issue by jcivjan** [instructure/canvas-lms#53]
I can't seem to use the Chat feature of the hosted Canvas version with the keyboard or with screen reader software. Am I missing something?
